### DISCUSSION TITLE
Weglot Translation Plugin support!!

### DISCUSSION THREAD

**Discussion by Uhtredtb** [inspireui/support#5298] [New Feature/Ideas Requests]
Please kindly make Weglot plugin compatible with inspire UI app.

It would be more convenient and flexible.

Here is the plugin URL:
https://wordpress.org/plugins/weglot/

**Comment by Uhtredtb** [comment]
Hi, is there any update? it's 2022 now no one haven't reply be back!
@minhcasi

**Comment by minhcasi** [comment]
@bambootb - this request is not enough voting so we have not started to integrate it yet, please try to check if you could customize the plugin to support lang param - example: `?lang=ar` then it will work out of the box
